src/engine/DnCManager.cpp

```diff
@@ -264,19 +263,19 @@ void DnCManager::printResult()
         break;
     }
     case DnCManager::UNSAT:
-        std::cout << "DnCManager::solve UNSAT query" << std::endl;
+        std::cout << "UNSAT" << std::endl;
         break;
     case DnCManager::ERROR:
-        std::cout << "DnCManager::solve ERROR" << std::endl;
+        std::cout << "ERROR" << std::endl;
         break;
     case DnCManager::NOT_DONE:
```

Comment:
is this needed now? since it is called after the DnCManager finishes....

Reply:
In the sequential Marabou, we also print out result string other than SAT, UNSAT, and UNKNOWN, such as ERROR and TIMEOUT.
https://github.com/guykatzz/Marabou/blob/c05a9a870b9deb5d5efbe0b2ae564f4598f5a4a8/src/engine/Marabou.cpp#L122

We might need to discuss what should Marabou print out as results as a whole.